### PR TITLE
Create config set url option

### PR DESCRIPTION
**Pull request by gabrielbussolo**
# Summary of this proposal

intends to fix the #145 issue

## Issue(s) number(s)

*Disclaimer: Please do not create a Pull Request without creating an issue first.*

## Checklist for submitter

- [ ] My PR has a related issue/bug number.
- [ ] My PR provides tests
  - [ ] Integration tests are added/passes
  - [ ] Unit tests are added/passes
  - [ ] End-to-end tests added/passes
  - [ ] Static code check added/passes
  - [ ] Linting on documentation added/passes
  - [ ] Does not affect code coverage stats
- [ ] My PR requires updating dependencies
- [ ] My PR has the documentation changes required.

## Checklist for reviewer

- [ ] The proposal fixes a bug/issue or implements a new feature that is well described.
- [ ] The proposal has sufficient test cases that covers the changes.
  - [ ] If changes an API, it does not break backwards compatibility (-1 major version).
- [ ] If integration is required, the proposal has integration tests.
- [ ] The proposal does not break coverage stats
- [ ] The proposal has the required documentation changes.

## Backport

- [ ] Backport to the latest stable release.

### CODE REVIEW

File: cmd/calyptia/config_url.go [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+	"os"
+	"path/filepath"
+)
+
+var errUrlNotFound = errors.New("url not found")
+
+func newCmdConfigSetURL(config *config) *cobra.Command {
+	return &cobra.Command{
+		Use:   "set_url URL",
+		Short: "Set the default project url so you don't have to specify it on all commands",
```
Comment (nicolasparada) [inline cmd/calyptia/config_url.go:18]:
```suggestion
		Short: "Set the default cloud URL so you don't have to specify it on all commands",
```

File: cmd/calyptia/config_url.go [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+			config.baseURL = url
+			return nil
+		},
+	}
+}
+
+func newCmdConfigCurrentURL(config *config) *cobra.Command {
+	return &cobra.Command{
+		Use:   "current_url",
+		Short: "Get the current configured default project url",
```
Comment (nicolasparada) [inline cmd/calyptia/config_url.go:35]:
```suggestion
		Short: "Get the current configured default cloud URL",
```

File: cmd/calyptia/config_url.go [hunk 3]
```diff
@@ -0,0 +1,137 @@
...
+			fmt.Fprintln(cmd.OutOrStdout(), config.baseURL)
+			return nil
+		},
+	}
+}
+
+func newCmdConfigUnsetURL(config *config) *cobra.Command {
+	return &cobra.Command{
+		Use:   "unset_url",
+		Short: "Unset the current configured default project url",
```
Comment (nicolasparada) [inline cmd/calyptia/config_url.go:46]:
```suggestion
		Short: "Unset the current configured default cloud URL",
```

File: cmd/calyptia/config_url.go [hunk 4]
```diff
@@ -0,0 +1,137 @@
...
+			if err != nil {
+				return err
+			}
+			config.baseURL = defaultCloudURLStr
+			return nil
+		},
+	}
+}
+
+func saveUrl(url string) error {
```
Comment (nicolasparada) [inline cmd/calyptia/config_url.go:58]:
```suggestion
func saveURL(url string) error {
```

File: cmd/calyptia/config_url.go [hunk 5]
```diff
@@ -0,0 +1,137 @@
...
+
+	err = os.WriteFile(fileName, []byte(url), fs.ModePerm)
+	if err != nil {
+		return fmt.Errorf("could not store file %q: %w", fileName, err)
+	}
+
+	return nil
+}
+
+func deleteSavedUrl() error {
```
Comment (nicolasparada) [inline cmd/calyptia/config_url.go:86]:
```suggestion
func deleteSavedURL() error {
```

File: cmd/calyptia/config_url.go [hunk 6]
```diff
@@ -0,0 +1,137 @@
...
+
+	err = os.Remove(fileName)
+	if err != nil {
+		return fmt.Errorf("could not delete default project url: %w", err)
+	}
+
+	return nil
+}
+
+func savedUrl() (string, error) {
```
Comment (nicolasparada) [inline cmd/calyptia/config_url.go:110]:
```suggestion
func savedURL() (string, error) {
```

**Review by niedbalski — APPROVED** [review]

**Review by nicolasparada — APPROVED** [review]
Did some fixes so now it works

**Pull request merged**